web_client/templates/layout/headerImage.pug

```diff
@@ -25,8 +25,14 @@ button.btn.btn-default.navbar-btn.h-open-annotated-image(type='button')
   | #[span.icon-list] Annotated images...
 button.btn.btn-default.navbar-btn.h-open-image(type='button')
   | #[span.icon-picture] Open image...
+if !previousImageLink || !nextImageLink
```

Comment:
In pug, you can make this whole thing a single javscript block with `-` .  For instance,
```
-
  var titlePre = '', titleNext = '';
  if (previousImageLink && nextImageLink) {
    titlePre = 'Previous image - ' + previousImageName;
    titleNext = 'titleNext='Next image - ' + nextImageName;
  }
```

Reply:
Oh, I see. Thank you and I will change it.